File: Daily/2023/1732_findTheHighestAltitude.cs
public class Solution {
    public int LargestAltitude(int[] gain) {
        int n = gain.Length;
        int max_alt = 0, cur_alt = 0;

        for (int idx = 0; idx < n; idx++) {
            cur_alt += gain[idx];
            max_alt = Math.Max(
                max_alt,
                cur_alt
            );
        }
    
        return max_alt;
    }
}

File: Categories/Array/1732_findTheHighestAltitude.py
from typing import *

class Solution:
    def largestAltitude(self, gain: List[int]) -> int:
        n = len(gain)

        max_alt, cur_alt = 0, 0
        for idx in range(n):
            cur_alt += gain[idx]
            max_alt = max(
                max_alt,
                cur_alt
            )
        
        return max_alt